Cargo.toml

```diff
@@ -61,12 +66,15 @@ midnight-storage_v6          = { git = "https://github.com/midnightntwrk/midnigh
 midnight-transient-crypto_v6 = { git = "https://github.com/midnightntwrk/midnight-ledger", package = "midnight-transient-crypto", tag = "ledger-6.1.0-alpha.5" }
 midnight-zswap_v6            = { git = "https://github.com/midnightntwrk/midnight-ledger", package = "midnight-zswap", tag = "ledger-6.1.0-alpha.5" }
 nix                          = { version = "0.30" }
+once_cell                    = { version = "1.19" }
```

Comment:
Is once_cell really required? I thought most of the functionality was merged into std now.
(E.g. https://github.com/rust-lang/rust/pull/105587 )